lib/ansible/modules/packaging/os/rhn_channel.py

```diff
@@ -115,7 +127,15 @@ def main():
     password = module.params['password']
 
     # initialize connection
-    client = xmlrpc_client.ServerProxy(saturl)
+    if module.params['ca_cert'] is not None and sys.version_info > (2, 7, 9):
```

Comment:
If your claim below is correct, you need:
```suggestion
    if module.params['ca_cert'] is not None and sys.version_info >= (2, 7, 9):
```